src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="watch">
+      <div class="watch__center"></div>
+      <div class="watch__second-hand watch__second-hand--five-second watch__second-hand--sixty-second"></div>
```

Comment:
it would be better and for second hand
```suggestion
      <div class="watch__hand watch__hand--second"></div>
```